squad-dedicated-server.kvp

```diff
@@ -4,7 +4,7 @@ App.ApplicationPort2=27165
 App.ApplicationPort3=0
 App.ApplicationReadyMode=RegexMatch
 App.BaseDirectory=./squad-dedicated-server/403240/
-App.CommandLineArgs=MULTIHOME={{$ApplicationIPBinding}} Port={{$ApplicationPort1}} QueryPort={{$ApplicationPort2}} RCONPORT={{$RemoteAdminPort}} RCONPASSWORD={{$RemoteAdminPassword}} -log -fullcrashdump
+App.CommandLineArgs=MULTIHOME={{$ApplicationIPBinding}} Port={{$ApplicationPort1}} QueryPort={{$ApplicationPort2}} RCONPORT={{$RemoteAdminPort}} RCONPASSWORD={{$RemoteAdminPassword}} FIXEDMAXTICKRATE={{$MaxTPS}} -log -fullcrashdump
```

Comment:
Take off the $ on MaxTPS here and in the setting. That's reserved for AMP variables.

Reply:
Fixed